src/app/help/index.njk

```diff
@@ -177,7 +177,7 @@ GAR | Help
             </tr>
             <tr class="govuk-table__row">
               <td class="govuk-table__cell">Authority to carry refused. DO NOT BOARD.</td>
-              <td class="govuk-table__cell">Pilots, operators and agents will receive a NO BOARD response via a call and email when Authority to Carry (ATC) has been refused.</td>
+              <td class="govuk-table__cell">This response message will not be displayed on the screen. Pilots, operators and agents will be instructed not to board an individual (NO BOARD) via a call and email when Authority to Carry (ATC) has been refused.</td>
```

Comment:
"This response message will not be displayed on the screen" is it mentioned anywhere? And are we stopped showing the NO BOARD message on the status page?